spec/api-notification-spec.js

```diff
@@ -81,10 +85,10 @@ describe('Notification module', () => {
       }
     ]
 
-    assert.equal(n.actions[0].type, 'button')
-    assert.equal(n.actions[0].text, '3')
-    assert.equal(n.actions[1].type, 'button')
-    assert.equal(n.actions[1].text, '4')
+    expect(n.actions[0].type).to.equal('button')
```

Comment:
Shouldn't we check that `n.actions` is an array of the expected size first?

Reply:
done